### ISSUE TITLE
BUG?: mask has too few dimensions

### ISSUE THREAD

**Issue by andbiz** [Radiomics/pyradiomics#421]
Hi all

I encountered this Warning/Error while processing my data:

"mask has too few dimensions (number of dimensions 1, minimum required 1)"

The error is thrown by the function checkMask() in imageoperation.py:

```
if ndims **<=** minDims:
    logger.error('mask has too few dimensions (number of dimensions %d, minimum required %d)', ndims, minDims)
    return None, correctedMask
```

Should it be:
```
if ndims **<** minDims:
    logger.error('mask has too few dimensions (number of dimensions %d, minimum required %d)', ndims, minDims)
    return None, correctedMask
```
?

Thank you

**Comment by JoostJM** [comment]
Yes it should be. Thanks for the catch, I will update this in the code.

However, keep in mind that 1 dimension (ie, just a line of voxels) might indicate an 'invalid' ROI in many cases. I will therefore also update the default value to require a minimum of 2 dimensions (I.e. 2D segmentation), which you can still force back to 1 (or 0), as required, in the parameter file.

**Comment by JoostJM** [comment]
fixed by 623b836
